src/main/java/frc/robot/util/Vision.java

```diff
@@ -67,7 +67,7 @@ public ArrayList<EstimatedRobotPose> getEstimatedPoses(Pose2d referencePose) {
     for (int i = 0; i < m_cameras.size(); i++) {
       Optional<EstimatedRobotPose> estimatedPose = m_cameras.get(i).getEstimatedPose(referencePose);
       // If the camera can see an april tag, add it to the array list
-      if (estimatedPose.isPresent()) {
+      if (estimatedPose.isPresent() && estimatedPose.get()!=null &&estimatedPose.get().estimatedPose!=null) {
```

Comment:
do you really need to check that every bit is not a null, I am pretty sure if estimated Pose didn't exist than when you call it to get the pose it will spit out an error and "estimatedPose.isPresent()" and "estimatedPose.get()!=null" do the same thing.

Reply:
Sometimes, it tries to find April tags that don't exist, like 20, and I think those are present, but they don't return a valid pose.

Reply:
I also want to check if estimatedPose.get() is null because the next part of the line will crash the code if it's null, even if it should never be null.